Gaijinizer patch/Map499.txt

```diff
@@ -1,208 +1,259 @@
 ﻿> GAIJINIZER PATCH FILE
 > BEGIN STRING
 ｼｭﾙﾙﾙﾙﾙ……
-> CONTEXT: Map499/events/7/pages/0/25/Dialogue/other8 < UNTRANSLATED
-> CONTEXT: Map499/events/7/pages/0/129/Dialogue/other8 < UNTRANSLATED
-
+> CONTEXT: Map499/events/7/pages/0/25/Dialogue/other8
+> CONTEXT: Map499/events/7/pages/0/129/Dialogue/other8
+#CheckLater
 > END STRING
```

Comment:
Same with the previous one, we can use the sound we agreed on earlier: 
![image](https://user-images.githubusercontent.com/48231102/141700516-e9948b51-c10e-44ea-bd9e-5fda246ca653.png)